src/main/resources/net/sourceforge/pmd/util/fxdesigner/less/tests.less

```diff
@@ -65,6 +66,16 @@
   }
 }
 
+.list-view .list-cell,
+.list-view:focused .list-cell:selected {
```

Comment:
for unknown reasons, if I don't include selectors for `:focused` and `:selected` the size is reset to 24 when the row is selected… The only place where that size is set is in a very generic selector `.icon-button`, so I don't understand why it would take precedence…

Reply:
I don't either :shrug: